Sources/Thumbprint/Components/Pill.swift

```diff
@@ -16,9 +16,9 @@ open class Pill: UIView, UIContentSizeCategoryAdjusting {
         public static var defaultPadding: CGFloat = 12.0
         public static var defaultTextStyle: Font.TextStyle = .title7
 
-        var height: CGFloat
-        var padding: CGFloat
-        var textStyle: Font.TextStyle
+        private(set) var height: CGFloat
+        private(set) var padding: CGFloat
+        private(set) var textStyle: Font.TextStyle
```

Comment:
Shouldn't this be `public private(set)`?

Reply:
I didn't realize that was a thing, but yep that is what I was going for 👍